javascript/test/TripServiceSpec.js

```diff
@@ -0,0 +1,11 @@
+import assert from 'assert';
+import TripService from '../src/TripService';
+
+describe('TripService', () => {
+
+    it('should return no trips if users are not friends', () => {
+        const tripService = new TripService();
+        assert.equal(tripService.getTripsByUser('not a friend'), []);
+    });
+
+});
```

Comment:
Makes sense, just wanted to add a failing tests because javascript doesn't have any test framework that is as "standard" as e.g. JUnit so that folks know how to add tests. I've replaced it with a completely unrelated failing test.